src/containers/shared/components/Transaction/NFTokenBurn/test/NFTokenBurnSimple.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
nit: The folder name for this test is inconsistent with the other test folder (One is called `tests` and one is called `test`) - Not sure which one is normal, but we should standardize that.

Reply:
Great catch.  Looks like all other places it is called `test`.  The old location must have been a one off.

Reply:
Do we prefer `test` going forward?